### PR TITLE
n: add page

### PR DESCRIPTION
**Pull request by Vanessa219**
[n](https://github.com/tj/n) is a Node.js version management: no subshells, no profile setup, no convoluted API, just simple.

### CODE REVIEW

**Comment by Vanessa219** [comment]
@CLAassistant already linked.

File: pages/common/n.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# n
+
+> Node version management.
+> Node.js version management: no subshells, no profile setup, no convoluted API, just simple.
```
Comment (agnivade) [inline pages/common/n.md]: I see that you have copied this verbatim from the repo home page. We try to be as objective as possible. Unless, its really adding value to the first description, there's no need to add a second line.

File: pages/common/n.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# n
+
+> Node version management.
```
Comment (agnivade) [inline pages/common/n.md]: Maybe better to say - "Tool to manage multiple node versions". Node version management can mean a lot of things.

File: pages/common/n.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`n rm {{version ...}}`
+
+- Output versions installed and select the current version:
+
+`n`
+
+- Execute node version with [args ...]:
+
+`n use <version> [args ...]`
```
Comment (agnivade) [inline pages/common/n.md]: Please wrap version and args.. within tokens.

File: pages/common/n.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Output bin path for version:
+
+`n bin <version>`
```
Comment (agnivade) [inline pages/common/n.md]: Same as above.

**Comment by Vanessa219** [comment]
@agnivade I have modified it according to your request

**Comment by sbrl** [comment]
Wouldn't a symlink be better/ Then you'd only have to alter one copy to have it reflected in both - and git supports them too.

**Comment by agnivade** [comment]
We have an open issue about symlinks [here](https://github.com/tldr-pages/tldr/issues/787). Its a WIP. We can revisit this when that issue is resolved. Until then, file duplication is fine.

**Comment by agnivade** [comment]
Hey @Vanessa219 - I made some changes which I feel simplifies the page better. Let me know your thoughts.

**Comment by Vanessa219** [comment]
I think it's fine.  Thx. Pls merge it.

**Comment by agnivade** [comment]
@waldyrious / @jeeftor - ping.

File: pages/linux/n.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# n
+
+> Tool to manage multiple node versions.
+
+- Install a given version of node. If the version is already installed, it will be activated:
+
+`n {{version}}`
+
+- Display installed versions and activate any version:
```
Comment (sbrl) [inline pages/linux/n.md]: "activate any version" seems a bit confusing. Does it mean that it activates the latest version automatically if one hasn't been activated yet?

Comment (agnivade) [inline pages/linux/n.md]: No. It displays  list of versions and you can use the arrow keys to select a version and then activate the one you want.

Yea, I agree that part can be clarified.

Comment (waldyrious) [inline pages/linux/n.md]: "and interactively activate one of them:"?

Comment (agnivade) [inline pages/linux/n.md]: done

File: pages/linux/n.md [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 `n {{version}}`
 
-- Display installed versions and activate any version:
+- Display installed versions and interactively activate one of them:
```
Comment (sbrl) [inline pages/linux/n.md]: That's much better :smiley_cat:

**Pull request merged**